common/utils/index.ts

```diff
@@ -10,4 +10,5 @@ export {
   composeFinalQuery,
   removeBacktick,
 } from './query_utils';
-export { uiSettingsService } from './settings_service';
+
+export * from './core_services';
```

Comment:
Should we use `*` here? Is linting allowing it?

Reply:
lint didn't complain but I was going to remove functions in `core_services`. didn't find a way yet since embeddable is not rendered by observability plugin but also needs PPLService